### PR TITLE
Commitment language

### PR DESCRIPTION
**Pull request by BlakeTurner**

### CODE REVIEW

File: how-we-make-commitments.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# How we make commitments
+
+At Bench, we have lots of conversations. These conversations are frequently about how we want to improve things. So how do we make sure that things _actually happen_ as a result of these conversations? We make commitments.
+
+Making Commitments is not a natural phenomenon. It requires practice, and it's okay if it feels awkward at first. We've built this page as a reference to help you with your practice.
+
+## Commitment flowchart
+
+We've built a [flowchart](https://miro.com/app/board/o9J_lwSXARg=/) to illustrate how commitments are made at Bench. We recommend familiarizing yourselves with it, and then using this page as a reference for the terms themselves.
```
[the remark this answers is not in the document]
Comment (BlakeTurner) [reply how-we-make-commitments.md:9]: Good call. I made it public.

File: how-we-make-commitments.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# How we make commitments
+
+At Bench, we have lots of conversations. These conversations are frequently about how we want to improve things. So how do we make sure that things _actually happen_ as a result of these conversations? We make commitments.
+
+Making Commitments is not a natural phenomenon. It requires practice, and it's okay if it feels awkward at first. We've built this page as a reference to help you with your practice.
```
[the remark this answers is not in the document]
Comment (BlakeTurner) [reply how-we-make-commitments.md]: ✅

File: how-we-make-commitments.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+We've built a [flowchart](https://miro.com/app/board/o9J_lwSXARg=/) to illustrate how commitments are made at Bench. We recommend familiarizing yourselves with it, and then using this page as a reference for the terms themselves.
+
+## Terms
+
+**_Dissatisfaction_**: A general belief that things can be better than they are. In the real world, "dissatisfaction" often has negative connotations—but not at Bench. Dissatisfaction is the requisite first step for any change, big or small. Glory in it!
+
+**_Concern_**: Very closely related to dissatisfaction, but often related to the perceived downsides of what you are dissatisfied with. Often looks like "I am concerned that because of situation X, thing Y will happen." As an individual, you can analyze your concerns to find your dissatisfaction. As a leader or collaborator, you can listen to concerns and dig in to find the underlying dissatisfaction.
+
+**_Satisfaction_**: The opposite of dissatisfaction. You're happy with how things are. Just don't be satisfied to _too_ long.
```
Comment (vweller) [inline how-we-make-commitments.md]: The sentence does not make sens as is. 
You are either missing an "o" in one of the too's
OR
The word to should not be there.
Just don't be satisfied to _too_ long.

File: how-we-make-commitments.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Commitment flowchart
+
+We've built a [flowchart](https://miro.com/app/board/o9J_lwSXARg=/) to illustrate how commitments are made at Bench. We recommend familiarizing yourselves with it, and then using this page as a reference for the terms themselves.
+
+## Terms
+
+**_Commitment_**: Commitments are our best tool for talking about the future we're creating in concrete terms. They have two key components: "the thing I will do" and "by when". Each commitment is an individual choice—no one can "commit you to something" (well, other than to an asylum). It is something that _only you can do_. It is through individuals making powerful commitments about how they will shape the future that we build an incredible company. *Important:* Commitments are made based on our current understanding of the world. We receive new information every day, and often this information changes how we think about this future. **When this happens, we must update our commitments so they once again represent the future as we see it.** This is not failure, this is the system working as intended.
```
Comment (dmorgantini) [inline how-we-make-commitments.md:13]: > It is through individuals making powerful commitments about how they will shape the future that we build an incredible company.

I found this sentence difficult to parse; it makes sense but it's difficult to parse.

File: how-we-make-commitments.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+**_Dissatisfaction_**: A general belief that things can be better than they are. In the real world, "dissatisfaction" often has negative connotations—but not at Bench. Dissatisfaction is the requisite first step for any change, big or small. Glory in it!
+
+**_Concern_**: Very closely related to dissatisfaction, but often related to the perceived downsides of what you are dissatisfied with. Often looks like "I am concerned that because of situation X, thing Y will happen." As an individual, you can analyze your concerns to find your dissatisfaction. As a leader or collaborator, you can listen to concerns and dig in to find the underlying dissatisfaction.
+
+**_Satisfaction_**: The opposite of dissatisfaction. You're happy with how things are. (Just don't be satisfied to _too_ long.)
```
Comment (dmorgantini) [inline how-we-make-commitments.md:19]: I feel like this makes it seem binary, while the reality is I can be satisfied with one area and dissatisfied with another.

**Review by dmorgantini — APPROVED** [review]

**Comment by BlakeTurner** [comment]
Thanks all for the comments!

**Pull request merged**